src/cloud-api-adaptor/ibmcloud/ROKS_SETUP.md

```diff
@@ -0,0 +1,272 @@
+    ```
+
+1. Wait until the cluster is completely up and running before proceeding ...
+
+1. Get the `kubeconfig` for your cluster
+
+    ```bash
+    ibmcloud ks cluster config --cluster "$CLUSTER_NAME" --admin
+    ```
+
+### Configure an OpenShift cluster
+
+By default, your Red Hat OpenShift cluster will not work with the peer pod components. Using the environment variables set in the previous section, proceed with the following steps. 
+
+1. Add a cluster inbound security group rule for the `kata-agent` client
+
+    ```bash
+    export CLUSTER_ID=$(ibmcloud ks cluster get --cluster "$CLUSTER_NAME" --output json | jq -r .id)
+    export CLUSTER_SG="kube-$CLUSTER_ID"
+    export KATA_SG=$(ibmcloud is vpc "$VPC_ID" -json | jq -r .default_security_group.id)
+    ibmcloud is sg-rulec "$CLUSTER_SG" inbound udp --port-min 4789 --port-max 4789 --remote "$KATA_SG"
+    ```
+
+1. Label worker nodes for `cloud-api-adapter`
+
+    ```bash
+    kubectl label nodes $(kubectl get nodes -o jsonpath={.items..metadata.name}) node.kubernetes.io/worker=
+    ```
+
+1. Allow `cloud-api-adapter` to update pod finalizers
+
+    ```bash
+    kubectl apply -n default -f - <<EOF
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: openshift-caa-finalizer-role
+    rules:
+    - apiGroups:
+      - ""
+      resources:
+      - "pods/finalizers"
+      verbs:
+      - "update"
+    ---
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: openshift-caa-finalizer-role-binding
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: openshift-caa-finalizer-role
+    subjects:
+    - kind: ServiceAccount
+      name: cloud-api-adaptor
+      namespace: confidential-containers-system
+    EOF
+    ```
+
+1. Give `cc-operator` and `cloud-api-adapter` priviledged OpenShift SCC permission
+
+    ```bash
+    oc create namespace confidential-containers-system
+    oc project confidential-containers-system
+    oc adm policy add-scc-to-user privileged -z cc-operator-controller-manager
+    oc adm policy add-scc-to-user privileged -z cloud-api-adaptor
+    ```
+
+## Upload a PeerPod VM Custom Image
+
+A peer pod VM image needs to be created as a VPC custom image in IBM Cloud in order to create the peer pod instances
+from. You can do this by following the [image instructions in ./README.md](./README.md#peer-pod-vm-image), or run the following command to use a prebuilt demo image.
+
+Run the following command from the root directory of the `cloud-api-adapter` repository:
+
+```bash
+src/cloud-api-adaptor/ibmcloud/image/import.sh ghcr.io/confidential-containers/podvm-generic-ubuntu-amd64:ci-pr2162 "$REGION" --platform linux/amd64
```

Comment:
@stevenhorsman Is there an official build with the CRI-O pause command fix yet, that we can use here instead of this ghcr.io image?

Reply:
Yes, the latest build and 0.11.0 release builds should have that fix in now.

Reply:
I updated the image and made a few other small improvements in the doc.

I also ran through the whole set of steps (with the import.sh fix that just merged) and it works for me, so from my perspective it's good to go. PTAL.